frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -884,7 +884,9 @@ function ReaderRolling:onUpdatePos()
     self:updatePos()
 
     Device:setIgnoreInput(false) -- Allow processing of events (on Android).
-    UIManager:discardEvents(true) -- Discard events, which might have occured (double tap).
+    UIManager:discardEvents(0.2) -- Discard events, which might have occured (double tap).
```

Comment:
occurred

Reply:
I wonder if there's a "better" way than hardcoding .2? No objection.

Reply:
This 0.2 is a bit of a random choice, I just tested on my Kobo and emulator it is ok.
For the defaults when using "true", 0.8s on eInk, 0.4s otherwise, I did more tests and tuning because I knew what I wanted to avoid, dict window appearing and preventing accidental dismissals. In this case, it's less obvious what's the target :) it's usually a partial/ui refreshes that we may not even noticed. Using 0 feels may be a bit risky as the display will use some milliseconds, but using too much is too much.
So, no real "better" idea for a "better" value :)